docs/artifact-repository-ref.md

```diff
@@ -29,10 +33,12 @@ data:
 
 You can override the repository for a workflow as follows:
 
-```
+```yaml
 spec:
   artifactRepositoryRef:
-    key: minio
+    namespace: my-ns # default is the  workflow's namespace
+    configMap: my-cm # default is "artifact-repositories"
```

Comment:
I don't think we should allow a namespace resource to access configuration across a namespace. This will be the first time we've done this. I feel users could abuse this to reach into kube-system to get other config unrelated to workflows

Reply:
I'm going to design this out.